### DISCUSSION TITLE
如何修改版权文章链接文本

### DISCUSSION THREAD

**Discussion by StarWishsama** [reuixiy/hugo-theme-meme#362] [Q&A] [answered]
如题，因为我使用了 Nginx 转发，实际的链接不应该是这样的。meme 是否支持自定义文章链接文本呢？

![image](https://user-images.githubusercontent.com/25561848/144733951-6a81a16c-9afc-48af-a4e7-4444d478d1bc.png)

**Comment by reuixiy** [comment, marked as answer]
可以参考 https://gohugo.io/commands/hugo_server/

通过 `--baseURL` 参数指定 `baseURL`
